package.json

```diff
@@ -10,6 +10,7 @@
     "/dist"
   ],
   "dependencies": {
+    "@radix-ui/react-radio-group": "^0.1.0",
```

Comment:
i'm happy to see radix being used in the project now, i'll implement his usage in the modal and in others components that needs more acessibility, thanks for taking the lead and adding it ❤️